local-cli/server/findSymlinksPaths.js

```diff
@@ -6,7 +6,28 @@ module.exports = function findSymlinksPaths(lookupFolder) {
   const folders = fs.readdirSync(lookupFolder);
   const resolvedSymlinks = folders.map(folder => path.resolve(lookupFolder, folder))
     .filter(folderPath => fs.lstatSync(folderPath).isSymbolicLink())
-    .map(symlink => path.resolve(process.cwd(), fs.readlinkSync(symlink)));
+    .map(symlink => {
+      const symlinkCache = new Set();
+      let prevSymlink;
+      let nextSymlink = symlink;
+      while (true) {
+        if (symlinkCache.has(nextSymlink)) {
+          throw Error(`Symlink recursion detected:\n  ${nextSymlink}`);
```

Comment:
Truth. I'll just switch to using an `Array` and then use `slice` to get the relevant sequence.

Though it could get a little messy if the sequence is long.